### PR TITLE
Fix example links in Readme 

### PR DESCRIPTION
**Pull request by sbis04**
Some of the example links present in the `Readme.md` file was not correct, have fixed those.

### CODE REVIEW

**Comment by bharat-biradar** [comment]
@sbis04 Thank you for the PR =].

**Review by bharat-biradar — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -216,11 +214,7 @@ final result = await barcodeScanner.processImage(inputImage);
 final text = await textDetector.processImage(inputImage);
 ```
 
-**To know more see this [example](./example/lib/VisionDetectorViews/text_detector_view.dart)**
-
-## Face Detection
```
Comment (fbernaly) [inline README.md:221]: @sbis04: you deleted the Face Detection section I added.

File: README.md [hunk 2]
```diff
@@ -206,6 +203,7 @@ It returns List<[Barcode]()>
 ```
 final result = await barcodeScanner.processImage(inputImage);
 ```
+To know more see this [example](example/lib/VisionDetectorViews/barcode_scanner_view.dart)
```
Comment (fbernaly) [inline README.md:206]: Duplicated link

Comment (sbis04) [reply README.md:206]: Yes, I just noticed sending a new PR for it now.

File: README.md (hunk 1 above)
Comment (sbis04) [reply README.md:221]: I am not sure how did this happen, maybe while merging it occurred. I am sending a PR with it re-added.

**Comment by sbis04** [comment]
@fbernaly I have fixed these and have sent a new PR, please have a look at it: #18

**Pull request merged**